src/components/va-accordion/va-accordion-item.e2e.ts

```diff
@@ -0,0 +1,46 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('va-accordion-item', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent('<va-accordion-item></va-accordion-item>');
+    const element = await page.find('va-accordion-item');
+
+    expect(element).toEqualHtml(`
+    <va-accordion-item class="hydrated">
+      <mock:shadow-root>
+        <button aria-controls="content" aria-expanded="false"></button>
+        <div id="content">
+          <slot></slot>
+        </div>
+      </mock:shadow-root>
+    </va-accordion-item>`);
+  });
+
+  it('sets aria-expanded to true based on prop', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<va-accordion-item header="The header">Content inside</va-accordion-item>');
+    const component = await page.find('va-accordion-item');
+    const button = await page.find('va-accordion-item >>> button');
+
+    expect(button.getAttribute('aria-expanded')).toEqual('false');
+
+    component.setProperty('open', true);
+    await page.waitForChanges();
+
+    expect(button.getAttribute('aria-expanded')).toEqual('true');
+  });
+
+  it('fires a custom event when the button is clicked', async () => {
```

Comment:
Okay, so combine this with the above test, and we're ensuring that clicking fires the event, and that when the `open` property is set to `true`, it'll be opened like we expect. The part we're missing is where we stitch the two together, but that happens in the `va-accordion` wrapper.

I'm following. :+1: Might be good to make a comment either here or in the component itself saying what the purpose is of firing the event.